example_app_generator/generate_stuff.rb

```diff
@@ -137,8 +137,8 @@ def using_source_path(path)
           'config.warnings = false'
 gsub_file '.rspec', '--warnings', ''
 
-# Remove skips so we can test controller specs work
-gsub_file 'spec/controllers/gadgets_controller_spec.rb',
+# Remove skips so we can test that request specs work
+gsub_file 'spec/requests/gadgets_request_spec.rb',
```

Comment:
Could you explain why we have to change that?

Reply:
The file that was attempted to edit does not exist anymore. That's what I meant with my question in the PR body about *Integration / smoke test coverage*

Reply:
You need to ensure that the smoke coverage covers both scenarios.

Reply:
Done.

Also added generation with using the `--api` flag